Makefile

```diff
@@ -182,12 +182,27 @@ JQ = $(shell which jq)
 ifneq ($(VSCE),)
 ifneq ($(JQ),)
 EXT = $(shell vsce show --json deducteam.lambdapi 2>/dev/null | jq '.versions[0]' | jq '.version')
+
 .PHONY: publish-vscode-extension
 publish-vscode-extension:
 ifeq ($(EXT), $(shell cat editors/vscode/package.json | jq '.version'))
 	echo "extension already exists. Skip"
 else
+ifeq ($(PAT),)
```

Comment:
shouldn't it be VSCODE_PAT as mentioned in RELEASE.md ?

Reply:
VSCODE_PAT is the name of the secret in gitHub. PAT is the name of the env variable (as defined in the workflow file line 55)